src/psyclone/psyGen.py

```diff
@@ -5509,6 +5619,52 @@ def __init__(self, name, parent=None):
         self._name = name
         self._symbol_table = SymbolTable(self)
 
+    @staticmethod
+    def create(name, symbol_table, children):
+        '''Create a KernelSchedule instance given a name, a symbol table and a
+        list of child nodes.
+
+        :param str name: the name of the KernelSchedule.
+        :param symbol_table: the symbol table associated with this \
+            KernelSchedule.
+        :type symbol_table: :py:class:`psyclone.psyGen.SymbolTable`
+        :param children: a list of PSyIR nodes contained in the \
+            KernelSchedule.
+        :type children: list of :py:class:`psyclone.psyGen.Node`
+
+        :returns: a KernelSchedule instance.
+        :rtype: :py:class:`psyclone.psyGen.KernelInstance`
+
```

Comment:
docstring `:raises xxx:` missing

Reply:
Updated this and all other create methods.